workspaces/rbac/plugins/rbac/src/components/ConditionalAccess/ComplexConditionRow.tsx

```diff
@@ -254,7 +254,7 @@ export const ComplexConditionRow = ({
             color: theme => theme.palette.grey[500],
             flexGrow: 0,
             alignSelf: 'baseline',
-            marginTop: theme => theme.spacing(3.3),
```

Comment:
Can't we use 4?

`theme.spacing` makes a multiple of the spacing x the value here, or?

And its default is 8 which results in 33.6 pixels, or? Is it really worth to use half pixels here?

Reply:
If I use 4, it looks not vertically aligned with the input field. How do you think about `marginTop: '34px',` instead of `theme.spacing(4)`?

Reply:
I looked into the code and everything else would need a lot of refactoring. So we can go with that.